www/app/Services/SubAgentRunner.php

```
@@ -0,0 +1,241 @@
+    /**
+     * Resume an existing conversation by adding a new user message.
+     */
+    private function resume(
+        Agent $agent,
+        string $prompt,
+        bool $isBackground,
+        ExecutionContext $context,
+        string $conversationId,
+        ?string $workspaceId,
+    ): ToolResult {
+        $conversation = Conversation::where('uuid', $conversationId)
```

PHPStan (2.1.54)

[error] Call to an undefined static method App\Models\Conversation::where().


(staticMethod.notFound)